src/Pages/EmailPreferences/EmailPreferences.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Container,
+  Input,
+  Button,
+  Row,
+  Col,
+  Label,
+} from 'reactstrap';
+
+import Header from '../../Components/Header/Header';
+import { setUserEmailPreference, getUserData } from '../../APIFunctions/User';
+import './EmailPreferences.css';
+
+export default function EmailPreferencesPage(props) {
+  const BLUE = '#2d75eb';
+  const GREEN = '#1cc75b';
+  const RED = '#cf533a';
+  const [isOptedIntoEmails, setIsOptedIntoEmails] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [errorLoading, setErrorLoading] = useState(false);
+  const [userEmail, setUserEmail] = useState('');
+  const [buttonColor, setButtonColor] = useState(BLUE);
```

Comment:
similar nit to below, obv there's only one button but think about when u see the code in 1 month